packages/webapp/components/footer/FooterNavBarTabs.tsx

```diff
@@ -112,13 +112,15 @@ export function FooterNavBarTabs({
 
   const tabs = useMemo(() => {
     if (notificationsNavBar) {
-      mobileUxTabs.pop();
-      mobileUxTabs.splice(-1, 0, {
-        requiresLogin: true,
-        path: '/notifications',
-        title: 'Notifications',
-        icon: (active: boolean) => <Notifications active={active} />,
-      });
+      if (!mobileUxTabs.some((tab: FooterTab) => tab?.title === 'Activity')) {
```

Comment:
I still don't understand why would this appear multiple times? I know with the change now it wouldn't, but why in the first place?

Reply:
React... 🤷‍♂️

Reply:
I think it has to do with feature evaluation, since the `useMemo` depends on the feature, it gets re-triggered because of a "re-render" of useFeature somewhere somehow.